src/stories/Home.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Canvas } from '@storybook/addon-docs/blocks';
+
+<Meta title="Home" />
+
+# H5Web Component Library
+
+A library of React components and utilities for data visualization, based on [react-three-fiber](https://github.com/pmndrs/react-three-fiber).
+
+The components are organised in two categories:
+
+- 🎨 **Visualizations**: the top-level visualization components — i.e. the components you will most likely need.
+- 🧱 **Building Blocks**: the low-level components used by the visualization components — for advanced uses only.
+
+## Getting started
+
+1. Make you sure you have `react` and `react-dom` v16 or greater installed.
+1. Install the library:
+
+   ```bash
+   npm install @h5web/lib
+   ```
+
+1. Import and use a visualization component:
+
+   ```tsx
+   import { ScalarVis } from '@h5web/lib';
+
+   // ...
+
+   <ScalarVis value={1024} />;
+   ```
+
+1. Other visualization components expect to receive an `ndarray` and a domain as props.
+
+   1. Install `ndarray` and its corresponding types package (if you use TypeScript):
+
+      ```bash
+      npm install ndarray
+      npm install --save-dev @types/ndarray
+      ```
+
+   1. Pass an `ndarray` and domain to the visualization:
+
+      ```tsx
+      import ndarray from 'ndarray';
+      import { HeatmapVis, findDomain } from '@h5web/lib';
+
+      // Source nD array
+      const values = [
+        [0, 1, 2],
+        [3, 4, 5],
+      ];
+
+      // Flatten array
+      const flatArray: number[] = values.flat(Infinity);
+
+      // Convert to ndarray and find domain
+      const dataArray = ndarray<number>(values, [2, 3]);
```

Comment:
Should be `flatArray` here instead of `values`.

Reply:
🎣